mempool/mempool.go

```diff
@@ -661,6 +665,9 @@ func (mp *TxPool) RemoveDoubleSpends(tx *dcrutil.Tx) {
 // This function MUST be called with the mempool lock held (for writes).
 func (mp *TxPool) addTransaction(utxoView *blockchain.UtxoViewpoint,
 	tx *dcrutil.Tx, txType stake.TxType, height int64, fee int64) {
+	if txType == stake.TxTypeSSGen {
```

Comment:
how about, if 
```go
mp.cfg.OnVoteReceived != nil && txType == stake.TxTypeSSGen
```

and then remove the cfgCopy chunk from New()

Reply:
thanks, updating.

Reply:
Missed this earlier. Due to https://github.com/decred/dcrd/blob/7ca6163fb66a11650e69fa0985d31f1e990da8a9/mempool/mempool.go#L1629-L1632 The nil check isn't needed since `OnVoteReceived`'s default state is an empty anonymous func and not nil when the mempool is initialized.